src/5e-SRD-Monsters.json

```diff
@@ -40348,4 +40670,4 @@
     ],
     "url": "/api/monsters/zombie"
   }
-]
+]
```

Comment:
```suggestion
]

```

Reply:
I have no idea why VScode decided to do that lol

Reply:
It's probably a setting somewhere?

Reply:
@ragnork2 The setting in VS code is called "insert final newline". You should be able to search for it in the settings menu.

Reply:
ah kk thanks!